fms/utils/generation.py

```diff
@@ -22,8 +104,8 @@ def _make_cache_contiguous(past_key_value_states):
 
 def generate(
     model: Union[Callable, torch.nn.Module],
-    input_ids: torch.Tensor,
-    max_seq_len: int = 4096,
+    input_ids: Union[torch.Tensor, List[torch.Tensor]],
+    max_seq_len: int = 2048,
```

Comment:
I think most of our models are supporting at least 4096 now, is there a reason to reduce this?

Reply:
fixed in latest commit